### PR TITLE
docs(replays): Add API documentation for replay-counts endpoint

### PR DESCRIPTION
**Pull request by cmanallen**
Related: https://github.com/getsentry/team-replay/issues/353

### CODE REVIEW

**Review by michellewzhang — APPROVED** [review]
image

**Review by c298lee — APPROVED** [review]

File: src/sentry/replays/endpoints/organization_replay_count.py [hunk 1]
```diff
@@ -57,7 +76,21 @@ class OrganizationReplayCountEndpoint(OrganizationEventsV2EndpointBase):
...
+            GlobalParams.END,
+            GlobalParams.ENVIRONMENT,
+            GlobalParams.ORG_SLUG,
+            GlobalParams.START,
+            GlobalParams.STATS_PERIOD,
+            OrganizationParams.PROJECT,
+            VisibilityParams.QUERY,
+        ],
+        examples=RESPONSE_EXAMPLES,
+    )
```
Comment (michellewzhang) [inline src/sentry/replays/endpoints/organization_replay_count.py:91]:
```suggestion
    @extend_schema(
        operation_id="Return a count of replays for a given issue or transaction id.",
        parameters=[
            GlobalParams.END,
            GlobalParams.ENVIRONMENT,
            GlobalParams.ORG_SLUG,
            GlobalParams.START,
            GlobalParams.STATS_PERIOD,
            OrganizationParams.PROJECT,
            VisibilityParams.QUERY,
        ],
        examples=RESPONSE_EXAMPLES,
        tags=["Replays"]
    )
```

File: src/sentry/replays/endpoints/organization_replay_count.py [hunk 2]
```diff
@@ -57,7 +76,21 @@ class OrganizationReplayCountEndpoint(OrganizationEventsV2EndpointBase):
         }
     }
 
+    @extend_schema(
+        operation_id="Return a count of replays for a given issue or transaction id.",
```
Comment (michellewzhang) [inline src/sentry/replays/endpoints/organization_replay_count.py:80]:
```suggestion
        operation_id="Return a Count of Replays",
```

i believe this is the sidebar title, which are all capitalized and fairly short/concise on the existing docs

File: src/sentry/replays/endpoints/organization_replay_count.py [hunk 3]
```diff
@@ -28,6 +30,22 @@
 
 FILTER_HAS_A_REPLAY = "AND !replayId:''"
 
+RESPONSE_EXAMPLES = [
+    OpenApiExample(
+        "Query replay count by issue or transaction id",
+        description="Returns an object of id, count pairs.",
```
Comment (michellewzhang) [inline src/sentry/replays/endpoints/organization_replay_count.py:36]: description is not needed here

File: src/sentry/replays/endpoints/organization_replay_count.py [hunk 4]
```diff
@@ -28,6 +30,22 @@
 
 FILTER_HAS_A_REPLAY = "AND !replayId:''"
 
+RESPONSE_EXAMPLES = [
```
Comment (michellewzhang) [inline src/sentry/replays/endpoints/organization_replay_count.py:33]: let's move this example into `examples/replay_examples.py` ex https://github.com/getsentry/sentry/blob/5822d6b40bc7952360f3083ef9d33f8ede2f41a1/src/sentry/apidocs/examples/replay_examples.py#L44

File: src/sentry/replays/endpoints/organization_replay_count.py [hunk 5]
```diff
@@ -57,7 +76,21 @@ class OrganizationReplayCountEndpoint(OrganizationEventsV2EndpointBase):
...
+        parameters=[
+            GlobalParams.END,
+            GlobalParams.ENVIRONMENT,
+            GlobalParams.ORG_SLUG,
+            GlobalParams.START,
+            GlobalParams.STATS_PERIOD,
+            OrganizationParams.PROJECT,
+            VisibilityParams.QUERY,
+        ],
+        examples=RESPONSE_EXAMPLES,
```
Comment (michellewzhang) [inline src/sentry/replays/endpoints/organization_replay_count.py:90]:
```suggestion
        examples=ReplayExamples.GET_SELECTORS,,
```

Comment (michellewzhang) [inline src/sentry/replays/endpoints/organization_replay_count.py:90]: we're also missing a field here for `responses`

https://github.com/getsentry/sentry/blob/a27e8f061aa3f984f3a08af5fd684ac97974a457/src/sentry/replays/endpoints/organization_replay_index.py#L51

**Comment by cmanallen** [comment]
Looks like I messed up a merge.  Sorry for the pings.

**Pull request closed without merging**